scalafix-rules/src/main/scala/scalafix/internal/rule/ProcedureSyntax.scala

```diff
@@ -1,17 +1,28 @@
 package scalafix.internal.rule
 
+import metaconfig.Configured
+
 import scala.meta._
 import scala.meta.tokens.Token
 import scala.meta.tokens.Token.Equals
 import scala.meta.tokens.Token.RightParen
-
 import scalafix.util.Trivia
 import scalafix.v1._
 
 class ProcedureSyntax extends SyntacticRule("ProcedureSyntax") {
 
   override def description: String =
-    "Replaces deprecated procedure syntax with explicit ': Unit ='"
+    "Replaces deprecated procedure syntax with explicit ': Unit ='" +
+      "This rule is specific to scala 2, since procedure syntax is not supported in Scala 3"
+
+  override def withConfiguration(config: Configuration): Configured[Rule] = {
+    if (config.scalaVersion.startsWith("3"))
+      Configured.error(
+        """|This rule is specific to scala 2, since procedure syntax is not supported in Scala 3
+          |To fix this error, remove ProcedureSyntax from .scalafix.conf""".stripMargin
+      )
```

Comment:
other rules don't seem to use multi-line error messages, not sure if that can be a problem for some integrations?